book/src/windows_install.md

```diff
@@ -0,0 +1,70 @@
+# Installing GTK4 on Windows
+
+Installing GTK4 for gtk4-rs on Windows just takes a few minutes.
+Follow the **preparation** first and then proceed with installing GTK either for the **GNU toolchain**
+or the **MSVC toolchain**.
+
+## Preparation
+
+1. Install Rust using [rustup](https://rustup.rs)
+2. Install MSYS2 from [www.msys2.org](https://www.msys2.org/) 
+
+### Update path environment variable
+
+1. Go to settings -> Search and open `Advanced system settings` -> click on `Environment variables`
+2. Select `Path` -> Click on `Edit` -> Add the following three entries:
+ 
+```
+C:\msys64\mingw64\include
+C:\msys64\mingw64\bin
+C:\msys64\mingw64\lib
+```
+
+
+## GNU toolchain
+
+Using the GNU toolchain is usually the easier method and recommended unless you know you want to use MSVC instead.
```

Comment:
But it doesn't give you integration into basically all development tooling (debuggers, profilers, etc) that people usually use on Windows.

Reply:
Debugging should work fine with gdb + vscode. Not sure about profilers.

Reply:
Fair enough.

I though wonder if people will care if they can‘t use visual studio anyways.